py_experimenter/experimenter.py

```diff
@@ -185,9 +290,33 @@ def execute(self, approach, max_experiments: int = -1, random_order=False) -> No
         logging.info("All executions finished")
 
     def get_table(self) -> pd.DataFrame:
+        """
+        Returns the database table as `Pandas.DataFrame`. 
+
+        :return: The database table as `Pandas.DataFrame`. 
+        :rtype: pd.DataFrame
+        """
+        
         return self._dbconnector.get_table()
 
-    def execution_wrapper(self, approach, custom_fields: dict, keyfields, result_processor: ResultProcessor):
+    def execution_wrapper(self, approach: Callable[[dict, dict, ResultProcessor]], custom_fields: dict, keyfields: dict, result_processor: ResultProcessor):
+        """
```

Comment:
I would somehow expect this to tell me when I need the wrapper, and when I can just `execute`

Reply:
The method is used inside of `execute` and handles the actual execution of the approach. This should actually be an internal method, as it is not supposed to be used by a user. Therefore I added a `_` at the beginning of the method name. 

That said, do you suggest any specific changes in the documentation?